### DISCUSSION TITLE
More information on the app repositories list

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#107539] [Mobile]
### Select Topic Area

Product Feedback

### Body

- [ ] type of license
- [ ] forks
- [ ] issues
- [ ] pulls
- [ ] last update date
- [ ] tag topics
- [ ] type of project whether it is public or private
- [ ] update graph
- [ ] repository sorting

Website:
<img width="1100" alt="Screenshot 2024-02-11 alle 14 57 04" src="https://github.com/community/community/assets/20476002/fd841793-8df8-412e-8f15-8f8f88617ca8">

App android:
![image](https://github.com/community/community/assets/20476002/ac70992f-1020-4eea-a4f0-704a3b2a492e)

**Comment by HenSquared** [comment]
HI @Angelk90 , thanks for your feedback! I've created an internal issue to track this feedback. We will discuss internally and if there's any updates, we'll let ya know here.

**Reply by Angelk90** [reply]
@HenSquared : I hope it doesn't take too long to find out some information, for anything tag me and I will support you as I can.
